src/main/java/com/coherentsolutions/training/aqa/java/api/zhavrid/util/HttpClientUtil.java

```diff
@@ -0,0 +1,105 @@
+package com.coherentsolutions.training.aqa.java.api.zhavrid.util;
+
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import java.util.logging.Logger;
+
+public class HttpClientUtil {
```

Comment:
I see a duplicating parts like
try (CloseableHttpClient httpClient = HttpClients.createDefault())
...
can we move this part into a private method?

Reply:
and move that to `http` package